config/config.go

```diff
@@ -92,3 +92,28 @@ func NewDefaultConfig() *api.PharmerConfig {
 		},
 	}
 }
+
+func GetKmsFile() string {
```

Comment:
should it be in new file? can this stay in `.pharmer/config.d/config` file?

Reply:
No, it can't be in `.pharmer/config.d/config` file. `pharmer config get-contexts` returns all configs under `config.d`. It's not related to those configs.

Reply:
can you add a field for this url in here? `https://github.com/pharmer/pharmer/blob/master/apis/v1alpha1/config.go#L54-L59`